src/styles/blocks/stars/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  max-width: 100px;
+
+  & :nth-child(1n + 1) {
```

Comment:
remove redundant whitespace

Reply:
@tiserett  ))) I was asked before to put more spaces for better reading)